.github/workflows/main.yaml

```diff
@@ -16,3 +16,20 @@ jobs:
 
     - name: Run tests
       run: nix run -c ./run-tests.py
+
+  integration_tests:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install Nix
+      uses: cachix/install-nix-action@v12
+      with:
+        nix_path: nixpkgs=channel:nixos-20.09
+
+    - name: Run tests
+      run: |
+        mkdir -p ~/.config/nixpkgs/
+        echo '{ allowUnfree = true; }' > ~/.config/nixpkgs/config.nix
+        nix run -c ./integration-tests/test-nixpkgs.py -f '<nixpkgs>'
```

Comment:
Looks like it takes about 40 minutes to run on Github Actions, and returns the following list of attrs in 20.09 that cause crashes:

```
['acl', 'alertmanager-bot', 'attr', 'bash', 'binutils-unwrapped', 'bzip2', 'coreutils', 'coreutils-full', 'coreutils-prefixed', 'datadog-agent', 'diffutils', 'findutils', 'gawkInteractive', 'gcc-unwrapped', 'glibc', 'gnugrep', 'gnupatch', 'gnused', 'gnutar', 'gzip', 'holochain-go', 'javaPackages.junit_4_12', 'javaPackages.mavenHello_1_0', 'javaPackages.mavenHello_1_1']
```